libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -401,6 +401,7 @@ data HiddenPerm
     | ViewLegalHoldTeamSettings
     | ChangeLegalHoldUserSettings
     | ViewLegalHoldUserSettings
+    | DisableLegalHold
```

Comment:
My plan was to use `ChangeLegalHoldUserSettings` for that (change == add or remove).  If you want to keep `DisableLegalHold`, you probably also want to rename the others to make it consistent again.